### PR TITLE
Add Prime_E RGB PCB

### PR DESCRIPTION
**Pull request by holtenc**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/primekb/prime_e/config.h [hunk 1]
```diff
@@ -21,10 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0x5052
-#define PRODUCT_ID      0x0051
-#define DEVICE_VER      0x0001
 #define MANUFACTURER    PrimeKB
-#define PRODUCT         Prime_E
 #define DESCRIPTION     Ergo_45
```
Comment (zvecr) [inline keyboards/primekb/prime_e/config.h:25]:
```suggestion
```

File: keyboards/primekb/prime_e/std/config.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+/* USB Device descriptor parameter */
+#define PRODUCT_ID      0x0051
+#define DEVICE_VER      0x0001
+#define PRODUCT         Prime_E
+
+#define BACKLIGHT_PIN B7
+#define BACKLIGHT_LEVELS 5
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 8
```
Comment (zvecr) [inline keyboards/primekb/prime_e/std/config.h:30]:
```suggestion
```

File: keyboards/primekb/prime_e/config.h [hunk 3]
```diff
@@ -34,21 +31,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-#define BACKLIGHT_LEVELS 5
-
-
 /* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
 #define DEBOUNCE 5
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
 #define LOCKING_SUPPORT_ENABLE
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
```
Comment (zvecr) [inline keyboards/primekb/prime_e/config.h:44]:
```suggestion
#define LOCKING_RESYNC_ENABLE

#define DYNAMIC_KEYMAP_LAYER_COUNT 8
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by holtenc** [comment]
@zvecr committed all suggested changes.

**Review by zvecr — APPROVED** [review]

File: keyboards/primekb/prime_e/rgb/readme.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Prime_E
+
+![Prime_E](https://imgur.com/7Rl4JOA.jpg)
+
+An ergonomic 45%. 
+
+*Keyboard Maintainer: [Holtenc](https://github.com/holtenc/)  
+*Hardware Supported:Prime_E RGB PCBs, ATmega32u4
+*Hardware Availability: [Store Link](https://www.primekb.com)
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/rgb/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Holtenc](https://github.com/holtenc/)
* Hardware Supported:Prime_E RGB PCBs, ATmega32u4
* Hardware Availability: [Store Link](https://www.primekb.com)
```

File: keyboards/primekb/prime_e/rgb/readme.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    *make primekb/prime_e/rgb:default
+    *make primekb/prime_e/rgb:via
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/rgb/readme.md:14]:
```suggestion
    make primekb/prime_e/rgb:default
```

File: keyboards/primekb/prime_e/rgb/rules.mk [hunk 6]
```diff
@@ -0,0 +1,3 @@
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/rgb/rules.mk:3]:
```suggestion
BACKLIGHT_ENABLE = no
RGBLIGHT_ENABLE = yes
```

File: keyboards/primekb/prime_e/rules.mk [hunk 7]
```diff
@@ -23,11 +23,11 @@ COMMAND_ENABLE = no        # Commands for debug and configuration
...
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = yes            # USB Nkey Rollover
-BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality on B7 by default
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 MIDI_ENABLE = no            # MIDI support
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/rules.mk:31]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/primekb/prime_e/std/readme.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Prime_E
+
+![Prime_E](https://imgur.com/7Rl4JOA.jpg)
+
+An ergonomic 45%. 
+
+*Keyboard Maintainer: [Holtenc](https://github.com/holtenc/)  
+*Hardware Supported: Prime_E Standard PCBs, Atmega32u4
+*Hardware Availability: [Store Link](https://www.primekb.com)
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/std/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Holtenc](https://github.com/holtenc/)
* Hardware Supported: Prime_E Standard PCBs, Atmega32u4
* Hardware Availability: [Store Link](https://www.primekb.com)
```

File: keyboards/primekb/prime_e/std/readme.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    *make primekb/prime_e/std:default
+    *make primekb/prime_e/std:via
```
Comment (fauxpark) [inline keyboards/primekb/prime_e/std/readme.md:14]:
```suggestion
    make primekb/prime_e/std:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**